src/components/Features.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+class Features extends React.Component {
+    render() {
+        return <div className="features">
+            <h1>Features include:
+                <ul>
+                    <li>displaying a math problem</li>
+                    <li>showing a possible answer</li>
+                    <li>showing correct math answers</li>
+                    <li>helping people learn math</li>
+                </ul>
+            </h1>
+        </div>
+    }
+}
+
+export default Features;
```

Comment:
I see that this is a class component. Would it be better to do this as a functional component since it is only displaying some text?